### DISCUSSION TITLE
Is it possible to have a 3 or 5 grid column in Carbon?

### DISCUSSION THREAD

**Discussion by DenisCor** [carbon-design-system/carbon#12930] [Help] [answered]
Hi, is there a way to implement a 3 grid column with Carbon? Noticed in the docs it says we shouldn't use a 3 or 5 grid. Why? Is there a workaround for this?   This is strange imo..

**Comment by tay1orjones** [comment, marked as answer]
Yes, you can override the custom properties powering the grid if you would prefer to use a different value, but there's no guarantee of it being fully compatible with nested subgrids, narrow/condensed, etc as it's not an established pattern of use that is supported.

The use of a 16 column grid is a strategic design and branding decision rooted in adhering to the [2x grid](https://www.ibm.com/design/language/2x-grid) as defined by the foundational standards defined in the IBM Design Language.
